src/hooks/useDefaultWallpaper.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react'
+import { useClient, generateWebLink } from 'cozy-client'
+
+import DefaultWallpaper from 'assets/images/default-wallpaper.svg'
+
+export const useDefaultWallpaper = (): string | undefined => {
+  const client = useClient()
+  const [wallpaper, setWallpaper] = useState<string | undefined>(undefined)
+
+  useEffect(() => {
+    if (client) {
+      const defaultWallpaperUrl = generateWebLink({
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
+        cozyUrl: client.getStackClient().uri,
+        pathname: DefaultWallpaper,
+        hash: '',
+        searchParams: [],
+        slug: 'home',
+        // @ts-expect-error subdomain is not typed
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        subDomainType: client.getInstanceOptions().subdomain
+      })
+
+      setWallpaper(defaultWallpaperUrl)
+    }
+  }, [client])
```

Comment:
You do not need a hook for this no ?

Reply:
+1 is this not working just by using `generateWebLink` in `BackgroundContainer` comp?